python/ml4ir/base/model/relevance_model.py

```diff
@@ -48,7 +48,11 @@ def __init__(
         freeze_layers_list: list = [],
         compile_keras_model: bool = False,
         output_name: str = "score",
+        aux_output_name: str = "aux_score",
```

Comment:
I feel we could've created a subclass for `RelevanceModelAuxLoss`. Bad name suggestion on my part, but something along those lines

Reply:
I believe that would be a nicer design going forward if we need to have N objectives. I believe now this might complicate things at the moment.